packages/features/bookings/lib/handleNewBooking.ts

```diff
@@ -1839,7 +1887,7 @@ async function handler(
   }
 
   // After polling videoBusyTimes, credentials might have been changed due to refreshment, so query them again.
-  const credentials = await refreshCredentials(organizerUser.credentials);
+  const credentials = await refreshCredentials([...organizerUser.credentials, ...teamCredentials]);
```

Comment:
The logic downstream is going to pick the latest added credential first and thus there is no visibility and direct control on which credential would be picked. 

So, from a user perspective if there is Zoom app installed on Team, Org, and organizer level and the eventType for the organizer is booked, the meeting can go to any of the Team, Org, organizer connected accounts and it would keep on varying depending on which was installed most recently.


I think we should provide a way to be very clearly select which credential to use for an App for an EventType